tests/TestCase/Log/LogTest.php

```diff
@@ -399,6 +400,7 @@ public function testScopedLogging(): void
     /**
      * Test scoped logging backwards compat
      */
+    #[WithoutErrorHandler]
```

Comment:
I still think those should be FQCN inside the attribute instead of use statements.

Reply:
Indeed the code sniffer does this